src/Microsoft.AspNetCore.SignalR.Common/Internal/Encoders/LengthPrefixedTextMessageWriter.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Buffers;
+using System.Globalization;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.AspNetCore.SignalR.Internal.Encoders
+{
+    public static class LengthPrefixedTextMessageWriter
+    {
+        private const int Int32OverflowLength = 10;
```

Comment:
This is the max number of digits an Int32 number can have. (btw. this is not new code - it's the old TextMessageParser moved here and renamed, same with LengthPrefixedTextMessageParser)